### DISCUSSION TITLE
Proof

### DISCUSSION THREAD

**Discussion by Martin7-1** [courses-at-nju-by-hfwei/discrete-math-problem-sets#8] [Q&A] [answered]
请问老师：
**Question**: 如何怎么证明在任何一个图中的**奇数度数的顶点**是偶数个呢？(绿皮书P348页的定理)

**Comment by hengxin** [comment, marked as answer]
@Martin7-1 因为所有顶点的度数之和为边数的两倍，是偶数。见作业 hw9 第一题。

**Reply by Martin7-1** [reply]
明白了，谢谢老师。请问在考试中可以直接使用在作业中证过的定理吗？

**Comment by hengxin** [comment]
@Martin7-1 简单的可以直接用。如果复杂的话，需要写出过程。当然，简单、复杂也是相对的。

**Reply by Martin7-1** [reply]
好的，谢谢老师。
